### PR TITLE
Add support for Android Studio

### PR DESCRIPTION
**Pull request by attilah**
The PR adds support for Android Studio

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -109,6 +109,18 @@ const allEditors = () => [
...
 		],
 		keywords: []
+	},
+	{
+		id: 'androidstudio',
+		name: 'Android Studio',
+		binary: 'studio',
+		isTerminalEditor: false,
+		paths: [
+			'/Applications/Android Studio.app/Contents/MacOS/studio'
```
Comment (sindresorhus) [inline index.js]: This needs to include the paths on Windows and Linux too.

File: index.js [hunk 2]
```diff
@@ -109,6 +109,18 @@ const allEditors = () => [
 			'/Applications/Emacs.app/Contents/MacOS/Emacs'
 		],
 		keywords: []
+	},
+	{
+		id: 'androidstudio',
```
Comment (sindresorhus) [inline index.js]:
```suggestion
		id: 'android-studio',
```

I do realize I haven't been consistent about this.

File: readme.md [hunk 3]
```diff
@@ -7,16 +7,17 @@ This module is used by [`open-editor`](https://github.com/sindresorhus/open-edit
...
+- NeoVim
+- Sublime Text
 - TextMate
 - Vim
-- NeoVim
-- IntelliJ
-- GNU nano
-- GNU Emacs
+- Visual Studio Code
+- WebStorm
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes. This should be added to the bottom.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (attilah) [reply index.js]: Other entries could also have that, I was not sure about it, added for Android Studio.

File: index.js (hunk 2 above)
Comment (attilah) [reply index.js]: Done.
